_apps/rtd-bot.md

```diff
@@ -0,0 +1,27 @@
+---
+# A human-friendly name of your listing
+title: Read The Docs
+# A short description of what your app does
+description: Assists reviewing using Read The Docs
+# The slug of your hosted app on GitHub (https://github.com/apps/YOUR-SLUG)
+slug: rtd-bot
+# Include a few screenshots that show your app in action
+screenshots:
+- https://raw.githubusercontent.com/KengoTODA/rtd-bot/master/screenshot.png
+# The GitHub usernames of anyone who authored the app
+authors:
+- KengoTODA
+# The repository where the code is located
+repository: KengoTODA/rtd-bot
+# The address where this app is deployed
+host: https://rtd-bot.herokuapp.com
+---
+
+Get URL of documents when review changes in docs directory.
+
+Work with Read The Docs, then you'll find that PR for documentation needs additional steps like:
```

Comment:
> then you'll find that PR for documentation needs additional steps like:

Where does this documentation live?

Reply:
In Github repo, we have it in `docs` directory. In Read the Docs, it exists under specific domain like https://${PROJECT_NAME}.readthedocs.io/${LANG}/${BRANCH_NAME}/

Reply:
Yes, but my question is, how would a user figure that out just from your comment there? I was not able to figure it out :(

Reply:
Do you mean how user can find the URL of generated document? If so, it'll be injected into PR body like this:

https://raw.githubusercontent.com/KengoTODA/rtd-bot/master/screenshot.png